### ISSUE TITLE
Could not open the k8055 (port: 0)

### ISSUE THREAD

**Issue by sebastianzillessen** [rm-hull/k8055#6]
Hi there, 

I have an issue using this library with my `VM110N` (which is the same as the `K8055`, the only difference is that is shipped in a plug & play version, so you don't have to build it all together).

The installation succeeds, but then I get the following error during the start of the application:

`````
sudo k8055 -debug -p:0 -d:255
Vellmann Device found @ Address 003 Vendor 0x010cf Product ID 0x05500
Get driver name: - could not get bound driver: No data available
Found interface 0
Took over the device
Read retry
Read retry
Read retry
Could not open the k8055 (port:0)
Please ensure that the device is correctly connected.
`````

Any ideas where this could come from?
I already tried to use the K8055 drivers on a windows computer and there it worked.

Thanks for your help!

**Comment by rm-hull** [comment]
Do you get any relevant diagnostics from `dmesg` that you could post here?
Also how do you know `-p:0` is correct?

**Comment by sebastianzillessen** [comment]
Hi there, 

I just fixed it and created a pull request: #7 .

**Comment by rm-hull** [comment]
ok, great stuff, I'll have a look and get it merged in later on today
